src/OrchardCore.Modules/OrchardCore.Cors/Controllers/AdminController.cs

```diff
@@ -23,26 +23,23 @@ public class AdminController : Controller
         private readonly IAuthorizationService _authorizationService;
         private readonly CorsService _corsService;
         private readonly INotifier _notifier;
-        private readonly IStringLocalizer T;
-        private readonly IHtmlLocalizer<AdminController> TH;
+        private readonly IHtmlLocalizer<AdminController> H;
 
         public AdminController(
             IShellHost shellHost,
             ShellSettings shellSettings,
             IAuthorizationService authorizationService,
-            IStringLocalizer<AdminController> stringLocalizer,
-            IHtmlLocalizer<AdminController> htmlLocalizer,
             CorsService corsService,
-            INotifier notifier
+            INotifier notifier,
+            IHtmlLocalizer<AdminController> htmlLocalizer,
```

Comment:
I think you should remove the comma at the end of this line ;)

Reply:
This a last ctor parameter, not a last property initializer

Reply:
My internet is too bad today, I add it accidently during the merge ;)